.circleci/config.yml

```diff
@@ -354,6 +354,7 @@ jobs:
     executor: node-browsers-small
     steps:
       - run: *shallow-git-clone
+      - run: sudo corepack enable
```

Comment:
Does it with with just `corepack enable` sans sudo?

Reply:
Nope, this is what happens without sudo https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/84179/workflows/a00c10d8-5c25-4604-9332-5a71d51e512f/jobs/3025527